llvm/lib/Target/AIE/AIE2GenFixupInstrInfo.td

```diff
@@ -1012,40 +1012,48 @@ let Itinerary = II_VINSERT in {
 
 // 8.1.2 VEXTRACT - Extract scalar from vector
 let Itinerary = II_VEXTRACT in {
-  let Uses = [crVaddSign] in {
-    def VEXTRACT_D8 :
-      AIE2_vec_extract_inst_mv</*sign*/0b0, (outs OP_mMvSclDst:$dst), (ins mXm:$s1, OP_mRS4m:$idx),
-                              "vextract.d8", "$dst, $s1, $idx">,
+  let ItineraryRegPairs = [ItinRegClassPair<II_VEXTRACT_RS,[OperandRegClass<0, eR>]>, 
+                           ItinRegClassPair<II_VEXTRACT_P,[OperandRegClass<0, eP>]>, 
+                           ItinRegClassPair<II_VEXTRACT_M,[OperandRegClass<0, eM>]>,
+                           ItinRegClassPair<II_VEXTRACT_RS,[OperandRegClass<0, eS>]>, 
+                           ItinRegClassPair<II_VEXTRACT_DC,[OperandRegClass<0, eDC>]>,
+                           ItinRegClassPair<II_VEXTRACT_DJ,[OperandRegClass<0, eDJ>]>, 
+                           ItinRegClassPair<II_VEXTRACT_DN,[OperandRegClass<0, eDN>]>] in {
```

Comment:
I'd be happy if you can avoid un-necessary line changes, that makes the diff really hard to read.
Maybe do not add extra indentation and have:
```
let Itinerary = II_VEXTRACT in
let ItineraryRegPairs = [ItinRegClassPair<II_VEXTRACT_RS,[OperandRegClass<0, eR>]>, 
...
ItinRegClassPair<II_VEXTRACT_DN,[OperandRegClass<0, eDN>]>] in {

 // Unchanged

```

Reply:
(Basically, same as you did in the next commit 😉)